README.md

```diff
@@ -412,6 +419,14 @@ To work on the `bento_web` repository within a BentoV2 environment, run the foll
 This will clone the `bento_web` repository into `./repos/web` if necessary, and start it in development mode,
 which means on-the-fly Webpack building will be available.
 
+⚠️ **Warning for local dev** ⚠️
+
+In local mode, be sure to navigate to the cloned repository `./repos/web/` (or any other service repo you want to work on localy), 
+and checkout on the PR branch from which the dev Docker image was built. 
+
+You can find the default image tag variables in `./etc/bento.env` and overwrite them in `local.env`, look for the pattern `BENTOV2_(service name)_VERSION`. 
+
+The version tags correspond to the PR **number** (not its name), e.g. `BENTOV2_WEB_VERSION=pr-216` indicates that the image was built from PR #216 in bento_web.
```

Comment:
There is a way to checkout localy to a PR number without knowing the branch name, but the resulting local branch is in detached mode. Pretty hacky, but we would not need to find the right branch and to manually checkout after switching to local mode on a service.

Example given `BENTOV2_WEB_VERSION=pr-216`, where 216 is the pr number:
```
cd ./repos/web
git fetch origin pull/216/head && git checkout FETCH_HEAD
```

Reply:
I think I prefer manually letting people control the repo... detached mode might lead to other issues